README.md

```diff
@@ -1,13 +1,10 @@
 # Happy Thoughts
```

Comment:
I think once you come back to this project and make it work, would be nice to update the ReadMe files, so they look complete if you would like to include this project in your portfolio ✨